lib/cf/dashboard/src/lib/controllers/CfAbacusController.js

```diff
@@ -0,0 +1,26 @@
+'use strict';
+
+const logger = require('../logger');
```

Comment:
Have you considered the use of abacus-debug module? This might be a next step (issue + additional PR), but using it gives us the ability to change the debug level without restarting the app by exposing an endpoint.